lib/util.js

```diff
@@ -21,22 +21,24 @@ function getPaths(publicPath, compiler, url) {
 
     for (let i = 0; i < compilers.length; i++) {
       compilerPublicPath = compilers[i].options
-    && compilers[i].options.output
-    && compilers[i].options.output.publicPath;
-
-      if (compilerPublicPath.indexOf('/') === 0) {
-        compilerPublicPathBase = compilerPublicPath;
-      } else {
-        // handle the case where compilerPublicPath is a URL with hostname
-        compilerPublicPathBase = parse(compilerPublicPath).pathname;
-      }
+        && compilers[i].options.output
+        && compilers[i].options.output.publicPath;
```

Comment:
strange whitespace goings on here. the formatting of the original code looks off by a tab-space, but please try to avoid modifying whitespace in PRs.